src/cli.ts

```diff
@@ -75,7 +75,9 @@ async function buildDirectory(
   outDirPath: string,
   options: CLIOptions,
 ): Promise<void> {
-  const extension = options.sucraseOptions.transforms.includes("typescript") ? ".ts" : ".js";
+  const extensions = options.sucraseOptions.transforms.includes("typescript")
+    ? [".ts"]
```

Comment:
Could you add `.tsx` here as well?

Reply:
yeah, I thinked about and I was not sure